symfony/plugins/orangehrmOpenidAuthenticationPlugin/lib/authproviders/GoogleAuthProvider.php

```diff
@@ -23,43 +23,48 @@
  * Description of GoogleAuthProvider
  */
 class GoogleAuthProvider extends AbstractAuthProvider {
-
     const PROFILE_SCOPE = 'profile';
     const EMAIL_SCOPE = 'email';
+    protected $loginService = null;
+    protected $option= null;
```

Comment:
option is an object or string? 
if it's a string then default value should be ''

I feel this should be array and "requestcode" is one parm on it

Reply:
option is string type variable